crypto/rand/drbg_hash.c

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright 2021 The BabaSSL Project Authors. All Rights Reserved.
```

Comment:
It seems this should keep the original OpenSSL copyright

Reply:
Fixed